src/flutter/shell/platform/linux_embedded/window/elinux_window_wayland.h

```diff
@@ -181,6 +185,12 @@ class ELinuxWindowWayland : public ELinuxWindow, public WindowBindingHandler {
   // List of loaded Wayland cursor themes keyed by their (scaled) size.
   std::unordered_map<uint32_t, wl_cursor_theme*> wl_cursor_themes_;
 
+  // List of outputs that the window is currently rendered on.
+  std::vector<uint32_t> entered_outputs_;
```

Comment:
Is there any reason to use vector instead of set or unordered_set?

Reply:
Good point. I briefly looked over [Chromium's Wayland implementation](https://source.chromium.org/chromium/chromium/src/+/main:ui/ozone/platform/wayland/host/wayland_surface.h;l=374;drc=2e167900bce4951550b8d73b74a3368cd85181f8) and it was using a `vector` so I just did the same without giving it too much thought. But using an `unordered_set` definitely makes more sense here so I've replaced it with that.

Reply:
Thanks